modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -571,6 +563,14 @@ private DefaultDTOConverterContext _getDTOConverterContext(
 			contextUser);
 	}
 
+	private String _getFilterString() {
+		if (contextHttpServletRequest == null) {
+			return null;
+		}
+
+		return ParamUtil.getString(contextHttpServletRequest, "filter");
```

Comment:
Hey @ccorreagg, based on @matijapetanjek's message here 5c6f425eb98a3ed90a8c50d00dee5400c6ef3c4a, do you think we can use a method parameter instead of getting the filter from the request? we should not be using the request at all. If we use the method param it will be null in batch and that will be fine.

Reply:
Well, that would require a deeper analysis. As you may remember, the `Filter filter` parameter is injected via ContextProvider (FilterContextProvider), and in Object's, we are not currently managing Filters, but we are managing a String filter and we transform it into an Expression

Reply:
And the `Filter filter` parameter is being set by REST Builder, as there is a `String filter` parameter in the OpenAPI specification file

Reply:
ok let's leave it for now.